NuGet.Config

```diff
@@ -72,9 +167,12 @@
       <package pattern="vslangproj80" />
       <package pattern="vslangproj90" />
       <package pattern="vswebsite.interop" />
+      <package pattern="xunit" />
+      <package pattern="xunit.runner.visualstudio" />
+      <package pattern="Xunit.StaFact" />
```

Comment:
Like several other comments, these patterns already exist in the dotnet-public feed, and there's no logical reason why the VS SDK would list any of these packages as a dependency, let alone have some VS specific version that doesn't exist on the dotnet-public feed.